src/api/app/models/decision_favored_delete_user.rb

```diff
@@ -0,0 +1,61 @@
+class DecisionFavoredDeleteUser < Decision
+  after_create :create_event
+  after_create :delete_user
+
+  def description
+    'The moderator decided to favor the report and delete the user'
+  end
+
+  def self.display_name
+    'favored with user deletion'
+  end
+
+  def self.display?(reportable)
+    return false unless [Comment, BsRequest, User].any? { |c| reportable.is_a?(c) }
+
+    true
+  end
```

Comment:
IIUC it can be simplified with

```suggestion
  def self.display?(reportable)
    [Comment, BsRequest, User].any? { |c| reportable.is_a?(c) }
  end
```